src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+
+  &__image-block {
+    margin: auto;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Since the `&__image` is centered using `margin: auto`, there's no need to specify `margin-left` and `margin-right` individually. You can remove these lines to simplify the code.